demo/run.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 
+
+# Launches a snakefile of your choice in dry run mode (for debugging)
+# Usage: ./dry_run.sh "<snakemake_flags>" <snakefile.smk> <target_rule> 
+# Example: ./dry_run.sh example.smk example_all
+# snakefile.smk The snakefile you want to run
+# target_rule: The name of one of the target rules specified in one of the included Snakefiles
+# snakemake_flags: One or more flags for the snakemake to run, specified inside quotation marks
+
+
 # Default to all targets
-TARGETS=${@:-all}
```

Comment:
Why did you change the parameter order in this script?

Reply:
The reason for a change was because I had to update the run and dry-run files to accommodate the entry of a Snakefile in the command

Reply:
I understand. Please always update the usage statement to show the current working usage. You've changed how this script is run so you need to document it in the usage statement.